src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -245,6 +245,35 @@ def set_reboot_pending(self, is_reboot_pending):
         self.is_reboot_pending = is_reboot_pending
     # endregion
 
+    # region - Terminal state management
+    def report_sequence_number_changed_termination(self):
+        """ Based on the current operation, sends """
+        current_operation = self.execution_config.operation.lower()
+        error_code = Constants.PatchOperationErrorCodes.NEWER_OPERATION_SUPERSEDED
+        message = "Execution was stopped due to a newer operation taking precedence."
+
+        if self.execution_config.exec_auto_assess_only:
+            self.add_error_to_status(message, error_code, current_operation_override_for_error=Constants.ASSESSMENT)
+            self.set_assessment_substatus_json(status=Constants.STATUS_ERROR)
+            return
+
+        if current_operation == Constants.CONFIGURE_PATCHING.lower() or \
+                current_operation == Constants.CONFIGURE_PATCHING_AUTO_ASSESSMENT.lower() or \
+                current_operation == Constants.ASSESSMENT.lower() or \
+                current_operation == Constants.INSTALLATION.lower():
+            self.add_error_to_status(message, error_code, current_operation_override_for_error=Constants.CONFIGURE_PATCHING)
+            self.add_error_to_status(message, error_code, current_operation_override_for_error=Constants.CONFIGURE_PATCHING_AUTO_ASSESSMENT)
+            self.set_configure_patching_substatus_json(status=Constants.STATUS_ERROR)
+
+        if current_operation == Constants.ASSESSMENT.lower() or current_operation == Constants.INSTALLATION.lower():
```

Comment:
Remove installation check

Reply:
Assessment occurs during installation. It should stay shouldn't it?

Reply:
It should go to the substatus blob where the error originated. 
An example would be:
The overall operation is Installation, and an error occurs during Assessment, the error message with all context should go to PatchAssessmentSummary.

Since the overall operation was Installation, PatchInstallationSummary will also report error but it's error object will have a generic message stating, an error occurred during Assessment and to check PatchAssessmentSummary for error details. This part is handled in the code right now.